test/controller.test.ts

```diff
@@ -671,29 +671,6 @@ contract('Controller', ([owner, accountOwner1, accountOwner2, accountOperator1,
         )
       })
 
-      it('should revert withdrawing long otoken amount equal to zero', async () => {
-        const vaultCounter = new BigNumber(await controllerProxy.getAccountVaultCounter(accountOwner1))
-        assert.isAbove(vaultCounter.toNumber(), 0, 'Account owner have no vault')
```

Comment:
wait can we still have this check in the controller?

Reply:
This check is totally unneeded, will just consume more gas for the user that is withdrawing more than zero amount.